graphics/vtk/Portfile

```diff
@@ -132,6 +136,7 @@ foreach pyver ${python_versions} {
 
             configure.args-append \
                 -DVTK_WRAP_PYTHON:BOOL=ON \
+                -DVTK_INSTALL_PYTHON_MODULES_DIR:PATH=${frameworks_dir}/Python.framework/Versions/3.7/lib/python3.7/site-packages \
```

Comment:
I think this line should be:

`-DVTK_INSTALL_PYTHON_MODULES_DIR=${frameworks_dir}/Python.framework/Versions/${python_branch}/lib/python${python_branch}/site-packages`

Also, should there be a line:

`-DVTK_PYTHON_VERSION:STRING=3`

It may be a legacy variable that doesn't do much now, but I'm not sure.

Reply:
Yes the second variable I don't think it's needed.

I'll fix the other one.